### PR TITLE
docs(core): add documentation for the error NG0955

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**
This commit adds detailed description for the error NG0955.

### CODE REVIEW

**Comment by JeanMeche** [comment]
Could you also the entry to `adev/src/content/reference/errors/overview.md` please ? 🙂

File: adev/src/content/reference/errors/NG0955.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+There is also performance penalty associated with duplicated keys - internally Angular must use more sophisticated and slower data structures while repeating over collections with duplicated keys.
+
+## Fixing the error
+
+Change the tracking expression such that it uniquely identifies an item in a collection. In the discussed example the correct track expression would use the unique `key` property (`item.key`):
+
+```typescript
+@Component({
+    template: `@for (item of items; track item.kay) {{{item.value}}}`,
```
Comment (JeanMeche) [inline adev/src/content/reference/errors/NG0955.md:29]:
```suggestion
    template: `@for (item of items; track item.key) {{{item.value}}}`,
```

**Review by thePunderWoman — APPROVED** [review]
reviewed-for: fw-core, angular-dev, public-api

**Review by atscott — APPROVED** [review]
reviewed-for: public-api

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit a4a82af865c1b08f6b6be3bf32046b5845ad2b2e.

**Pull request closed without merging**